bliss/predict.py

```diff
@@ -316,7 +316,7 @@ def plot_predict(
     est_plocs = np.array(est_cat.plocs.cpu())[0]
     est_tile = est_cat.to_tile_params(
         cfg.encoder.tile_slen,
-        cfg.simulator.prior.max_sources,
+        cfg.simulator.survey.prior_config.max_sources,
```

Comment:
Again, I feel rather strongly that they prior isn't part of the survey; it's part of the statistical model that we use to interpret the survey data.

Reply:
Done.